docs/admin-guide/provisioning/kind.md

```diff
@@ -52,31 +53,39 @@ karina ca generate --name ingress-ca --cert-path .certs/ingress-ca.crt --private
 
 ```
 
-#### 4) Provision the cluster in kind:
+### 4) Provision the cluster in kind:
 
 ```shell
 karina provision kind-cluster -c test-cluster.yml
 ```
 
-#### 5) Deploy the bare minimum configuration:
+### 5) Deploy the bare minimum configuration:
+This command will deploy following:
+- CRDs for all production runtime defined in the 
 
 ```shell
-karina deploy phases --base --dex --calico -c test-cluster.yml
+karina deploy phases --crds --base --dex --calico -c test-cluster.yml
 ```
 
-#### 6) Deploy everything else that may be configured:
+### 6) Deploy everything else that may be configured:
 
 ```shell
 karina deploy all
 ```
 
+### Cleanup
+Stop and delete the container running Kind with
+```shell
+docker stop test-cluster-control-plane
+docker rm test-cluster-control-plane
+```
 
 
 ## Troubleshooting
 
 KIND cluster creation issues can be debugged by specifying the `--trace` argument to `karina` during creation:
 
-```bash
+```Shell
```

Comment:
is this intentional?

Reply:
no, it shouldn't be capitalized, pushing a fix

Reply:
Whats the difference between bash vs shell in mkdocs?

Reply:
I didn't think about mkdocs when updated this. I just saw all code boxes above are using `shell`. I usually zsh so it's a bit nicer in markdown but I'm not sure about mkdocs. Just looked at what was generated by mkdocs (I assume) here, it doesn't look different between `shell` and `bash`: https://karina.docs.flanksource.com/admin-guide/provisioning/kind/